backend/go.mod

```diff
@@ -45,6 +46,7 @@ require (
 	go.opentelemetry.io/otel/trace v1.35.0
 	golang.org/x/term v0.30.0
 	gopkg.in/yaml.v2 v2.4.0
+	gotest.tools/v3 v3.4.0
```

Comment:
Can you please remove this dependency? I think your test can be written similar to the TestDecodeBase64JSON test in auth_test.go which doesn't require any extra third party dependencies (only uses stdlib things for testing).